source/Preferences.cpp

```diff
@@ -108,6 +112,7 @@ void Preferences::Save()
 	out.Write("scroll speed", scrollSpeed);
 	out.Write("view zoom", zoomIndex);
 	out.Write("vsync", vsyncIndex);
+	out.Write("auto aim", autoAimIndex);
```

Comment:
Why do we add an "auto aim" setting, when an "Automatic aiming" setting already existed (which seems to be replaced here)?